### ISSUE TITLE
[M2-P11] Migrate Wall Loss Tutorial and Strategy Notebooks

### ISSUE THREAD

**Issue by Gorkowski** [uncscode/particula#951]
## Description

Migrate wall loss tutorial and strategy notebooks to current API patterns. This batch includes 3 notebooks demonstrating wall loss fundamentals and geometry-specific strategies.

## Context

**Parent Plan:** `adw-docs/dev-plans/maintenance/M2-notebook-api-migration.md`
**Phase:** M2-P11 (Dynamics - Wall Loss, batch 1 of 2)

**Notebooks in this phase:**
1. `docs/Examples/Chamber_Wall_Loss/Notebooks/Wall_Loss_Tutorial.ipynb` - may already be current (verify)
2. `docs/Examples/Chamber_Wall_Loss/Notebooks/Spherical_Wall_Loss_Strategy.ipynb`
3. `docs/Examples/Chamber_Wall_Loss/Notebooks/Rectangular_Wall_Loss_Strategy.ipynb`

**Known Issue:** `Wall_Loss_Tutorial.ipynb` had reported API changes with `get_pure_vapor_pressure` signature - verify if still an issue.

**Value:**
- Updates wall loss examples to current API
- Enables CI validation
- Provides consistent patterns across wall loss tutorials

## Scope

**Estimated Lines Changed:** ~100 lines (across 3 notebooks)
**Complexity:** Small (batch of related tutorials)

**Files to Modify:**
- `docs/Examples/Chamber_Wall_Loss/Notebooks/Wall_Loss_Tutorial.ipynb`
- `docs/Examples/Chamber_Wall_Loss/Notebooks/Spherical_Wall_Loss_Strategy.ipynb`
- `docs/Examples/Chamber_Wall_Loss/Notebooks/Rectangular_Wall_Loss_Strategy.ipynb`

### Pre-Phase Validation (REQUIRED FIRST STEP)
- [ ] Run `run_notebook` on ALL 3 notebooks to identify which need updates
- [ ] Check `Wall_Loss_Tutorial.ipynb` for `get_pure_vapor_pressure` issues
- [ ] Document failures and API patterns needing migration

### API Migration (for notebooks that need it)
- [ ] Update builder/factory patterns per M2 migration patterns
- [ ] Fix any `get_pure_vapor_pressure` signature issues
- [ ] Verify no deprecated imports remain

### Validation
- [ ] ALL 3 notebooks pass `run_notebook` within timeout
- [ ] All code cells parse with `ast.parse()`
- [ ] Save executed outputs using `writeExecuted` parameter

### Documentation
- [ ] Verify descriptions are clear and accurate
- [ ] Ensure geometry-specific notebooks explain their use cases

### Validation Commands

```bash
# Pre-validation
run_notebook({notebookPath: "docs/Examples/Chamber_Wall_Loss/Notebooks/Wall_Loss_Tutorial.ipynb", timeout: 120})
run_notebook({notebookPath: "docs/Examples/Chamber_Wall_Loss/Notebooks/Spherical_Wall_Loss_Strategy.ipynb", timeout: 120})
run_notebook({notebookPath: "docs/Examples/Chamber_Wall_Loss/Notebooks/Rectangular_Wall_Loss_Strategy.ipynb", timeout: 120})

# Post-migration with output saving
run_notebook({notebookPath: "docs/Examples/Chamber_Wall_Loss/Notebooks/Wall_Loss_Tutorial.ipynb", timeout: 120, writeExecuted: "docs/Examples/Chamber_Wall_Loss/Notebooks/Wall_Loss_Tutorial.ipynb"})
# (repeat for each notebook)
```

### Wall Loss Builder Pattern (from AGENTS.md)

```python
import particula as par

strategy = par.dynamics.ChargedWallLossStrategy(
    wall_eddy_diffusivity=0.001,
    chamber_geometry="spherical",
    chamber_radius=0.5,
    wall_potential=0.05,
    wall_electric_field=0.0,
    distribution_type="discrete",
)

wall_loss = par.dynamics.WallLoss(wall_loss_strategy=strategy)
aerosol = wall_loss.execute(aerosol, time_step=1.0, sub_steps=2)
```

## Testing Strategy

This is a documentation-only change (notebooks). Testing consists of:
1. Pre-phase validation to identify issues
2. Post-migration validation with `run_notebook` tool
3. Syntax validation (ast.parse) for all code cells

## References

**Maintenance Plan:**
- `adw-docs/dev-plans/maintenance/M2-notebook-api-migration.md`

**Related Files:**
- `docs/Examples/Chamber_Wall_Loss/Notebooks/` - Wall loss notebooks
- `.github/workflows/notebooks.yml` - CI validation workflow
